packages/carequality-sdk/README.md

```diff
@@ -0,0 +1,37 @@
+# `carequality-sdk`
+
+SDK to interact with the Carequality directory - by Metriport Inc.
```

Comment:
let's add a link to the CQ directory docs? this implementation is based off of them, so would be good to give some context here